catalog/core/catalog-core-defaultvalues/src/test/java/ddf/catalog/data/defaultvalues/DefaultAttributeValueRegistryImplTest.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.defaultvalues;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static ddf.catalog.data.Metacard.POINT_OF_CONTACT;
+import static ddf.catalog.data.Metacard.TITLE;
+import static ddf.catalog.data.impl.BasicTypes.BASIC_METACARD;
+
+import java.io.Serializable;
+import java.util.Optional;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.Sets;
+
+import ddf.catalog.data.DefaultAttributeValueRegistry;
+
+public class DefaultAttributeValueRegistryImplTest {
+    private static final String BASIC_METACARD_NAME = BASIC_METACARD.getName();
+
+    private static final String OTHER_METACARD_NAME = "other";
+
+    private DefaultAttributeValueRegistry registry;
+
+    @Before
+    public void setUp() {
+        registry = new DefaultAttributeValueRegistryImpl();
+    }
+
+    @Test
+    public void testSetDefaultValue() {
+        registry.setDefaultValue(BASIC_METACARD_NAME, TITLE, "foo");
```

Comment:
You use the "foo", "bar" and "foobar" strings a lot. How about making them static final? And compute the concat version from the first two.